boards.txt

```diff
@@ -230,7 +230,41 @@ esp32wrover.menu.DebugLevel.verbose=Verbose
 esp32wrover.menu.DebugLevel.verbose.build.code_debug=5
 
 ##############################################################
-
+magicbit.name=MagicBit
```

Comment:
you need to move this below the PICO board. Top 3 boards are the official chips and modules from Espressif :)